ncm-spma/src/main/pan/components/spma/yum/schema.pan

```diff
@@ -37,6 +37,7 @@ type component_spma_yum = {
     "run" ? legacy_binary_affirmation_string # Run the SPMA after configuring it
     "userpkgs_retry" : boolean = true
     "userpkgs" ? legacy_binary_affirmation_string # Allow user packages
+    "extra_reposdirs" ? absolute_file_path[] # List of additional repo dirs to be included
```

Comment:
add comment as annotation. also add that the priority: indentical repos in the extra dir precede (or not?) the quattor generated ones.

Reply:
wrt name: just `reposdirs`? or `userreposdirs`?

Reply:
Just `reposdirs` seems fine, I'll make sure the annotation explains what happens with regards to the managed repodir. As far as precendence... I might have to experiment to see how yum behaves.

Reply:
Given the yum.conf option is `reposdir` I think `reposdirs` makes sense. The only downside is this is additional config to be added to `reposdir`, there is no way to remove the default? OTOH that seems like such a niche use case that it's OK?